falco/templates/_helpers.tpl

```diff
@@ -229,3 +229,70 @@ be temporary and will stay here until we move this logic to the falcoctl tool.
     - mountPath: /gvisor-config
       name: falco-gvisor-config
 {{- end -}}
+
+
+{{- define "falcoctl.initContainer" -}}
+- name: falcoctl-artifact-install
+  image: {{ printf "%s/%s:%s" .Values.falcoctl.image.registry .Values.falcoctl.image.repository .Values.falcoctl.image.tag }}
+  imagePullPolicy: {{ .Values.falcoctl.image.pullPolicy }}
+  args: 
+    {{- $str := "[\"artifact\", \"install\", " }}
+    {{- range $key, $value := .Values.falcoctl.artifact.install.args }}
+    {{- $str = print $str "\"" $value "\"" "," }}
+    {{- end }}
+    {{- $str = print $str "]" }}
+    {{- $str | nindent 4 }}
```

Comment:
This is the way I found to concatenate args. Don't know if there is something smarter that can be done here!

Reply:
This :point_down: should be more straightforward! :angel: 

https://github.com/falcosecurity/charts/blob/9b27a5925f4c8a4d0fd500a2e963d572e2cce767/falco/templates/pod-template.tpl#L93-L95